tooling/artifact_cli/src/execution.rs

```diff
@@ -127,6 +128,7 @@ pub fn check_witness(
                 if actual != expected {
                     return Err(CliError::UnexpectedReturn { expected, actual: Some(actual) });
                 }
+                println!("[{}] Circuit output: {actual:?}", circuit_name);
```

Comment:
It looks like this will only be printed if the `return` is set in the input `Prover.toml` file, ie. if `expected_return` is [not empty](https://github.com/noir-lang/noir/blob/9116b271b4569c575b5d8d90dacfee0ca9a4c097/tooling/artifact_cli/src/execution.rs#L122).

I'd do it at the same level as all the other `if`s in this method, or potentially outside of it in `save_and_check_witness`.

Reply:
ah yes, I was going to quickly. I will move it